astropy/table/table.py

```diff
@@ -1829,6 +1835,7 @@ def show_in_notebook(
         html += jsv.ipynb(tableid, css=css, sort_columns=sortable_columns)
         return HTML(html)
 
+    @deprecated_renamed_argument("jsviewer", None, "6.1", pending=True)
     def show_in_browser(
```

Comment:
`show_in_browser` is not related to Jupyter, and is not broken. It creates a standalone HTML file and was the original reason of using datatables etc., the notebook integration came later. 
Not sure if it's widely used though.

Reply:
True, but if we are going to remove all JS code, it *might* make sense to remove all the related bits. It should also help with resolving issues like https://github.com/astropy/astropy/issues/10420 and https://github.com/astropy/astropy/issues/11188

I know I'm being a bit bold here and marking it as a pending deprecation may tell us if there are people out there using this bit :D

Reply:
I agree `show_in_browser` is not Jupyter-related. What about splitting this into two PRs? I think deprecating `show_in_notebook` is the right call, and that we shouldn't replace it. The underpinnings of notebook change too often for us to support well in astropy. 

I'm less clear about `show_in_browser`. 

On the one hand, it works. On the other hand, it adds dependencies that we are bundling in `jsviewer.py` that are not update regularly by dependabot. If we are going to supply `show_in_browser` then it seems worth discussing separately from `show_in_notebook`.

At the end of the day, I lean towards deprecating `show_in_browser` too and seeing what the reaction from the community is...

Reply:
I find show_in_browser to be very useful fwiw, although I don't really use the jsviewer flag that often.